cmd/argocd/commands/admin/settings_rbac.go

```diff
@@ -200,26 +202,50 @@ argocd admin settings rbac can someuser create application 'default/app' --defau
 }
 
 // NewRBACValidateCommand returns a new rbac validate command
-func NewRBACValidateCommand() *cobra.Command {
+func NewRBACValidateCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 	var (
 		policyFile string
 	)
 
 	var command = &cobra.Command{
-		Use:   "validate --policy-file=POLICYFILE",
+		Use:   "validate --policy-file POLICYFILE [--namespace NAMESPACE]",
```

Comment:
As policy-file is also optional now, put it in square brackets.
```suggestion
		Use:   "validate [--policy-file POLICYFILE] [--namespace NAMESPACE]",
```

Reply:
Done